File: BAEKJOON/cpp/10768_특별한_날.cpp
#include <iostream>
using namespace std;

int month, day;

void input() {
	cin >> month >> day;
}

void solution() {
	if (month > 2) {
		cout << "After";
		return;
	}

	if (month == 2) {
		if (day == 18) {
			cout << "Special";
			return;
		}

		if (day > 18) {
			cout << "After";
			return;
		}
	}

	cout << "Before";
}

int main() {
	ios::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	input();
	solution();

	return 0;
}

File: BAEKJOON/java/10768_특별한_날.java
import java.io.*;

public class Main {
    public static int month, day;
    public static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

    public static void main(String[] args) throws IOException {
        input();
        solution();

        bw.flush();
        bw.close();
    }

    private static void input() throws IOException {
        month = Integer.parseInt(br.readLine());
        day = Integer.parseInt(br.readLine());
    }

    private static void solution() throws IOException {
        if (month > 2) {
            bw.write("After");
            return;
        }

        if (month == 2) {
            if (day == 18) {
                bw.write("Special");
                return;
            }

            if (day > 18) {
                bw.write("After");
                return;
            }
        }

        bw.write("Before");
    }
}